<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventSelectionDialog</class>
 <widget class="QDialog" name="EventSelectionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meggie - Epoch Creation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>807</width>
        <height>640</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>400</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBoxEvent">
               <property name="title">
                <string>Select events to include in epoch collection</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QLabel" name="labelEventID">
                      <property name="text">
                       <string>Event ID:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="comboBoxEventID"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QLabel" name="labelMask">
                      <property name="text">
                       <string>Mask</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="spinBoxMask">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="toolTip">
                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mask bit used for finding events. For example, if mask is set to 1, first bit is ignored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                      <property name="minimum">
                       <number>0</number>
                      </property>
                      <property name="maximum">
                       <number>9999</number>
                      </property>
                      <property name="value">
                       <number>0</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="labelName">
                      <property name="text">
                       <string>Event name:      </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEditName"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QPushButton" name="pushButtonAdd">
                      <property name="text">
                       <string>Add to list &gt;&gt;</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QGroupBox" name="groupBoxRejection">
               <property name="title">
                <string>Rejection parameters</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_31">
                    <item>
                     <widget class="QCheckBox" name="checkBoxGrad">
                      <property name="toolTip">
                       <string>Include or exclude grad channels</string>
                      </property>
                      <property name="text">
                       <string>Grad</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxGradReject_3">
                      <property name="prefix">
                       <string/>
                      </property>
                      <property name="suffix">
                       <string> fT/cm</string>
                      </property>
                      <property name="maximum">
                       <double>1000000000.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>100.000000000000000</double>
                      </property>
                      <property name="value">
                       <double>3000.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_33">
                    <item>
                     <widget class="QCheckBox" name="checkBoxMag">
                      <property name="toolTip">
                       <string>Include or exclude mag channels</string>
                      </property>
                      <property name="text">
                       <string>Mag</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxMagReject_3">
                      <property name="suffix">
                       <string> fT</string>
                      </property>
                      <property name="maximum">
                       <double>1000000000.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>100.000000000000000</double>
                      </property>
                      <property name="value">
                       <double>4000.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_32">
                    <item>
                     <widget class="QCheckBox" name="checkBoxEeg">
                      <property name="toolTip">
                       <string>Include or exclude eeg channels</string>
                      </property>
                      <property name="text">
                       <string>EEG</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxEEGReject_3">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="suffix">
                       <string> uV</string>
                      </property>
                      <property name="maximum">
                       <double>1000000000.000000000000000</double>
                      </property>
                      <property name="value">
                       <double>40.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_34">
                    <item>
                     <widget class="QCheckBox" name="checkBoxEog">
                      <property name="toolTip">
                       <string>Include or exclude eog channels</string>
                      </property>
                      <property name="text">
                       <string>EOG</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxEOGReject_3">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="suffix">
                       <string> uV</string>
                      </property>
                      <property name="maximum">
                       <double>1000000000.000000000000000</double>
                      </property>
                      <property name="value">
                       <double>250.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBoxEpoch">
               <property name="title">
                <string>Epoch collection</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="labelCollectionName">
                      <property name="text">
                       <string>Collection name:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEditCollectionName">
                      <property name="text">
                       <string>Epochs</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="labelTmin">
                      <property name="text">
                       <string>Start time:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxTmin">
                      <property name="suffix">
                       <string> s</string>
                      </property>
                      <property name="decimals">
                       <number>3</number>
                      </property>
                      <property name="minimum">
                       <double>-10.000000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>0.100000000000000</double>
                      </property>
                      <property name="value">
                       <double>-0.200000000000000</double>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="labelTmax">
                      <property name="text">
                       <string>End time:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="doubleSpinBoxTmax">
                      <property name="suffix">
                       <string> s</string>
                      </property>
                      <property name="decimals">
                       <number>3</number>
                      </property>
                      <property name="maximum">
                       <double>9.900000000000000</double>
                      </property>
                      <property name="singleStep">
                       <double>0.100000000000000</double>
                      </property>
                      <property name="value">
                       <double>0.500000000000000</double>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBoxStim">
                    <property name="toolTip">
                     <string>Include or exclude stim channels</string>
                    </property>
                    <property name="text">
                     <string>Include stim channel in collection</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBoxEventList">
             <property name="title">
              <string>List of events with given event ID</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QListWidget" name="listWidgetEvents">
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QPushButton" name="pushButtonSaveEvents">
                  <property name="text">
                   <string>Save events to file...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonReadEvents">
                  <property name="toolTip">
                   <string>You can add events from Excel files that are in form: name, sample, previous event id, current event id.</string>
                  </property>
                  <property name="text">
                   <string>Read events from xls file...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonClear">
                  <property name="text">
                   <string>Clear list</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonBatching">
       <property name="text">
        <string>Batching...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCreateEpochs">
       <property name="text">
        <string>Create epochs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>lineEditCollectionName</tabstop>
  <tabstop>doubleSpinBoxTmin</tabstop>
  <tabstop>doubleSpinBoxTmax</tabstop>
  <tabstop>checkBoxStim</tabstop>
  <tabstop>comboBoxEventID</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>checkBoxGrad</tabstop>
  <tabstop>doubleSpinBoxGradReject_3</tabstop>
  <tabstop>checkBoxMag</tabstop>
  <tabstop>doubleSpinBoxMagReject_3</tabstop>
  <tabstop>checkBoxEeg</tabstop>
  <tabstop>doubleSpinBoxEEGReject_3</tabstop>
  <tabstop>checkBoxEog</tabstop>
  <tabstop>doubleSpinBoxEOGReject_3</tabstop>
  <tabstop>listWidgetEvents</tabstop>
  <tabstop>pushButtonSaveEvents</tabstop>
  <tabstop>pushButtonReadEvents</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonCreateEpochs</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCreateEpochs</sender>
   <signal>clicked()</signal>
   <receiver>EventSelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>814</x>
     <y>730</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>EventSelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>730</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxGrad</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBoxGradReject_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMag</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBoxMagReject_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEeg</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBoxEEGReject_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEog</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBoxEOGReject_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>665</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>listWidgetEvents</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFileChooserDialog()</slot>
 </slots>
</ui>
